package.json

```diff
@@ -4,7 +4,7 @@
   "description": "This exercise will help you to have a better understanding of the correct structure of a web project.",
   "main": "index.js",
   "scripts": {
-    "start": "webpack-dev-server",
+    "start": "webpack-dev-server webpack.common.js",
```

Comment:
You should have a dedicated `prod` file and a dedicated `dev` file. The `common` file should not serve as the entry file for the configuration, but instead it should be used by the `dev` and `prod` configuration files.